### PR TITLE
Task1

### PR DESCRIPTION
**Pull request by sokalau**

### CODE REVIEW

File: Task1/src/com/epam/task1/main/Controller.java [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+        start();
+    }
+
+    private void start() {
+        try (BufferedReader reader = new BufferedReader(new InputStreamReader(System.in))) {
+            String input;
+            while (true) {
+                System.out.print(MENU);
+                input = reader.readLine();
+                if (input.equals("5")) break;
```
Comment (vitodevjav) [inline Task1/src/com/epam/task1/main/Controller.java]: it's not correct handle part input inside while loop and the other part in separate method.

File: Task1/src/com/epam/task1/main/Controller.java [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+                handleInput(input);
+            }
+        } catch (IOException e) {
+
+        }
+    }
+
+    private void handleInput(String input) {
+        switch (input) {
+            case "1":
```
Comment (vitodevjav) [inline Task1/src/com/epam/task1/main/Controller.java]: create enum for your input cases

File: Task1/src/com/epam/task1/model/Vegetable.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    public String toString() {
+        return "Vegetable" + "{calorific=" + calorific + ", weight=" + weight + "}";
+    }
+
+    @Override
+    public int compareTo(Vegetable vegetable) {
+        return Double.compare(weight, vegetable.weight);
+    }
+
+    public double getCalorific() {
```
Comment (vitodevjav) [inline Task1/src/com/epam/task1/model/Vegetable.java]: don't see any sense to have public getters and setters

File: Task1/src/com/epam/task1/main/Controller.java (hunk 1 above)
Comment (sokalau) [reply Task1/src/com/epam/task1/main/Controller.java]: i don't know how do better

**Pull request merged**